plugins/woocommerce/includes/admin/helper/class-wc-helper-updater.php

```diff
@@ -155,12 +155,46 @@ public static function transient_update_themes( $transient ) {
 	 * @return void.
 	 */
 	public static function setup_update_plugins_messages() {
+		$is_site_connected = WC_Helper::is_site_connected();
 		foreach ( WC_Helper::get_local_woo_plugins() as $plugin ) {
 			$filename = $plugin['_filename'];
-			add_action( 'in_plugin_update_message-' . $filename, array( __CLASS__, 'add_install_marketplace_plugin_message' ), 10, 2 );
+			if ( $is_site_connected ) {
+				add_action( 'in_plugin_update_message-' . $filename, array( __CLASS__, 'add_install_marketplace_plugin_message' ), 10, 2 );
+			} else {
+				add_action( 'in_plugin_update_message-' . $filename, array( __CLASS__, 'add_connect_woocom_plugin_message' ) );
+			}
 		}
 	}
 
+	/**
+	 * Runs on in_plugin_update_message-{file-name}, show a message to connect to Woo.com for unconnected stores
+	 *
+	 * @return void.
+	 */
+	public static function add_connect_woocom_plugin_message() {
+		$connect_page_url = add_query_arg(
+			array(
+				'page' => 'wc-admin',
+				'tab'  => 'my-subscriptions',
+				'path' => rawurlencode( '/extensions' ),
+			),
+			admin_url( 'admin.php' )
+		);
+
+		printf(
+			wp_kses(
+			/* translators: 1: Woo Update Manager plugin install URL */
+				__( ' <a href="%1$s">Connect your store</a> to Woo.com to update.', 'woocommerce' ),
```

Comment:
Should be `woocommerce.com`?

Reply:
woocommerce.com domain now changed to woo.com

Reply:
will update this to woocommerce.com because we've finished migrating domain back to wooommerce.com